src/main/webapp/babel-src/feedBackBox.js

```diff
@@ -0,0 +1,135 @@
+import React from "react";
+import Dialog from 'material-ui/Dialog';
+import FlatButton from 'material-ui/FlatButton';
+import TextField from 'material-ui/TextField';
+import Snackbar from 'material-ui/Snackbar';
+import {FEEDBACK_CHAR_LIMIT, FEEDBACK_SNACKBAR_AUTO_HIDE_DURATION, UI_STRINGS, INLINE_STYLES} from './util';
+
+export class FeedBackBox extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            charCtr : 0,
+            errorMsg : '',
+            feedbackMsg: '',
+            sendSuccess : false,
+            sendError : false
+        };
+
+        // this.updateTextField = this.updateTextField.bind(this);
+    }
+
+    updateTextField(event,value) {
+
+        if (value.length < FEEDBACK_CHAR_LIMIT) {
+            this.setState({
+               charCtr : value.length,
+               errorMsg : '',
+               feedbackMsg : value
+            });
+        } else {
+            this.setState({
+                charCtr : FEEDBACK_CHAR_LIMIT,
+                errorMsg : UI_STRINGS.FEEDBACK_CHAR_LIMIT_ERROR_MSG
+            })
+        }
+    }
+
+    sendFeedBackMsg() {
+        $.ajax({
+            type: "POST",
+            url: "api/feedback",
+            data: JSON.stringify({"message": this.state.feedbackMsg}),

```

Comment:
currently in `searchBox.js` we have:

``` data: JSON.stringify({"filter" : query}),```

are you sure you want me to remove quotation marks? @Davidster

Reply:
yes, I'm sure. The same change should eventually be made in the place you pointed out as well. I had the habit of putting the quotations like that in the past because I didn't know that they aren't needed.